src/js/tech/flash.js

```diff
@@ -13,6 +13,7 @@ import FlashRtmpDecorator from './flash-rtmp';
 import Component from '../component';
 import window from 'global/window';
 import assign from 'object.assign';
+import pkg from '../../../package.json';
```

Comment:
Also, this does mean we include the whole package.json inside of the output. We probably want to use something like https://www.npmjs.com/package/browserify-versionify instead.